recipes/iterativewgcna/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "iterativeWGCNA" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1ee490a1b7a0e1164fda08d38053a17f974f82f0237b8c2864bd12a28f3a8efb"
+
+build:
+  number: 0
```

Can it be ``noarch``?
```suggestion
  number: 0
  noarch: python
```